### PR TITLE
[NFC] Fix typos in llvm-mc doc

### PR DESCRIPTION
**Pull request by lamb-j**

### CODE REVIEW

**Review by boomanaiden154 — COMMENTED** [review]
Probably quite a few really minor nits for a small patch. Feel free to ignore them if you disagree with them. Just thought we could try and improve the clarity even a bit more while you're here.

Thanks for taking the time to help improve the documentation!

File: llvm/docs/CommandGuide/llvm-mc.rst [hunk 1]
```diff
@@ -11,13 +11,13 @@ SYNOPSIS
...
-architecture.
+The :program:`llvm-mc` command takes as input assembly code for a specified
+architecture and generates as output an object file or executable for a
+specified architecture.
 
-:program:`llvm-mc` provide powerful set of the tool for working with the
-machine code such as encoding of their instruction and their internal
-representation, disassemble string to bytes etc.
+:program:`llvm-mc` provides a powerful tool for working with machine code,
+such as encoding of instructions and displaying their internal representations,
```
Comment (boomanaiden154) [inline llvm/docs/CommandGuide/llvm-mc.rst:19]: Nit: of here seems redundant. Maybe just `encoding instructions`?

File: llvm/docs/CommandGuide/llvm-mc.rst [hunk 2]
```diff
@@ -11,13 +11,13 @@ SYNOPSIS
 DESCRIPTION
 -----------
 
-The :program:`llvm-mc` command take input as the assembly code for a specified
-architecture and generate object file or executable as a output for a specified
-architecture.
+The :program:`llvm-mc` command takes as input assembly code for a specified
```
[the remark this answers is not in the document]
Comment (lamb-j) [reply llvm/docs/CommandGuide/llvm-mc.rst]: good idea, added!

File: llvm/docs/CommandGuide/llvm-mc.rst [hunk 3]
```diff
@@ -11,13 +11,13 @@ SYNOPSIS
...
+architecture and generates as output an object file or executable for a
```
[the remark this answers is not in the document]
Comment (lamb-j) [reply llvm/docs/CommandGuide/llvm-mc.rst]: good idea, added!

**Pull request merged**